packages/frontend/src/pages/Templates/index.tsx

```diff
@@ -62,15 +64,17 @@ export default function Templates(): JSX.Element {
                 key={index}
                 icon={() => (
                   <Box bg="primary.100" p={2} borderRadius={4}>
-                    {<TemplateIcon iconName={template.iconName} />}
+                    <TemplateIcon iconName={template.iconName} />
                   </Box>
                 )}
                 onClick={() => navigate(URLS.TEMPLATE(template.id))}
               >
-                <Flex flexDir="column" gap={2} mt={2}>
-                  <Text textStyle="subhead-1">{template.name}</Text>
-                  <Text textStyle="body-2">{template.description}</Text>
-                </Flex>
+                <Skeleton isLoaded={!loading}>
+                  <Flex flexDir="column" gap={2} mt={2}>
+                    <Text textStyle="subhead-1">{template.name}</Text>
+                    <Text textStyle="body-2">{template.description}</Text>
+                  </Flex>
+                </Skeleton>
```

Comment:
this skeleton will never appear

Reply:
huh if i click to templates page i see it though